components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html

```diff
@@ -17,15 +17,15 @@
         <chef-heading>Profiles</chef-heading>
         <chef-subheading>Compliance profiles manage security and compliance scans.</chef-subheading>
 
-        <div class="profiles-search">
+        <div class="profiles-search" *ngIf="!profilesEmpty || isAvailableProfilesVisible">
```

Comment:
Do not show the search bar if there are no install profiles and the available profiles are not visible.